dali/dfu/dfuutil.cpp

```diff
@@ -421,8 +421,15 @@ class CFileCloner
             // JCSMORE: it may be this can replace the need for the other 'clone*' attributes altogether.
             if (srcfdesc->queryProperties().hasProp("_remoteStoragePlane"))
             {
-                attrs.setPropTree("cloneFromFDesc", createPTreeFromIPT(srcTree));
-                return;
+                if (srcdali && !srcdali->endpoint().isNull())
+                {
+                    attrs.setPropTree("cloneFromFDesc", createPTreeFromIPT(srcTree));
+                    StringBuffer host;
+                    attrs.setProp("@cloneFrom", srcdali->endpoint().getEndpointHostText(host).str());
```

Comment:
Should this be getEndpointHostText or getEndpointIpText?

Reply:
I think it's okay/correct as it is, but please let me know what your potential concern with hostname vs IP here?